container/dockerd-entrypoint.sh

```diff
@@ -6,6 +6,7 @@ source /home/model-server/mms_config.sh
 
 if [[ "$1" = "serve" ]]; then
     shift 1
+    mkdir -p /opt/ml/errors
```

Comment:
Sagemaker only mounts the error volumes for trusted service accounts in which case it will always be there, for containers in customer accounts we need to create this folder